Source/astcenc_compress_symbolic.cpp

```diff
@@ -1231,17 +1231,24 @@ float compress_symbolic_block(
 
 	float mode_cutoff = ctx.config.tune_block_mode_limit / 100.0f;
 
-	// next, test mode #0. This mode uses 1 plane of weights and 1 partition.
-	// we test it twice, first with a modecutoff of 0, then with the specified mode-cutoff.
-	// This causes an early-out that speeds up encoding of "easy" content.
+	// Trial using 1 plane of weights and 1 partition.
+
+	// Most of the time we test it twice, first with a mode cutoff of 0 and
+	// then with the specified mode cutoff. This causes an early-out that
+	// speeds up encoding of easy blocks. However, this optimization is
+	// disabled for 4x4 and 5x4 blocks where it nearly always slows down the
```

Comment:
Is that really 5x4 or should it be 5x5?

Reply:
Yes, should be 5x4 - ASTC has many non-square to get fine-grained bitrate steppings.